modules/modes/select.js

```diff
@@ -113,6 +113,31 @@ export function modeSelect(context, selectedIDs) {
         return parents;
     }
 
+    // find the child nodes for selected ways
+    function childNodeIdsOfSelection(onlyCommon) {
+        var graph = context.graph();
+        var childs = [];
+
+        for (var i = 0; i < selectedIDs.length; i++) {
+            var entity = context.hasEntity(selectedIDs[i]);
+
+            if (!entity || !['area', 'line'].includes(entity.geometry(graph))){
```

Comment:
If the selection includes a node and a way, should this method return nothing, or should it return the node and the way’s vertices?

Reply:
Good question. we could go either way but I would prefer to return nothing in this case mainly for consistency reasons:
- We are explaining this shortcut through 'way(s) selected' section so It seems more straightforward having effect only if ways are selected
- selectParent only works if vertices are selected so it would make sense for its antonym to have similar type of constraints.